src/handler.jl

```diff
@@ -1,6 +1,6 @@
 # Do not share a stream between processes
 # The token would be shared so putting would give InvalidSequenceTokenException a lot
-struct CloudWatchLogHandler{F<:Formatter} <: Handler{F, Union{}}
```

Comment:
This was throwing `LoadError: LoadError: too many parameters for type`

Reply:
This changed between Memento 0.12 and 0.13. If you want to support both, you'll need to make this conditional. I would just drop Memento 0.12.

Reply:
This changed between Memento 0.12 and 0.13. If you want to support both, you'll need to make this conditional. I would just drop Memento 0.12.